src/lib.rs

```diff
@@ -155,16 +161,18 @@ impl<'a> Nmea {
         }
     }
 
-    /// Returns used sattelites
-    pub fn satellites(&self) -> Vec<Satellite> {
-        let mut ret = Vec::with_capacity(20);
+    /// Returns used satellites
+    pub fn satellites(&self) -> Vec<Satellite, 58> {
```

Comment:
It would be nice if 58 become constant with comment why exactly 58.
And it would be nice to replace 15 with constant time expression that calculate it from 58 in SatsPack.

Reply:
I don't know what the limits are for these 2 values tbh and I feel more comfortable gradually improving them.
I feel that the code might get more complex especially with the public API and those constant values.

I think we can improve the API a lot and see how we can do the merging of the messages you mentioed.